aeron-client/src/main/java/io/aeron/logbuffer/LogBufferDescriptor.java

```diff
@@ -908,4 +925,48 @@ public static int computeAssembledFrameLength(final int length, final int maxPay
 
         return HEADER_LENGTH + (numMaxPayloads * maxPayloadSize) + remainingPayload;
     }
+
+    /**
+     * Get the value of the socket receive buffer length (rcvBuf) for this log.
+     *
+     * @param metadataBuffer containing the meta data.
+     * @return the value of the receive buffer length (rcvBuf) used for this log.
+     */
+    public static int rcvbufLength(final UnsafeBuffer metadataBuffer)
+    {
+        return metadataBuffer.getInt(LOG_RCVBUF_LENGTH_OFFSET);
+    }
+
+    /**
+     * Set the value of the socket receive buffer length for this log.
+     *
+     * @param metadataBuffer containing the meta data.
+     * @param rcvBuf         value to be set.
+     */
+    public static void rcvbufLength(final UnsafeBuffer metadataBuffer, final int rcvBuf)
+    {
+        metadataBuffer.putInt(LOG_RCVBUF_LENGTH_OFFSET, rcvBuf);
+    }
+
+    /**
+     * Get the value of the send buffer length for this log.
+     *
+     * @param metadataBuffer containing the meta data.
+     * @return the value of the send buffer length.
+     */
+    public static int sndbufLength(final UnsafeBuffer metadataBuffer)
```

Comment:
Should be named the same as in other places, i.e. `socketSndbufLength`.

Reply:
Done